bc_obps/reporting/models/json_schema.py

```diff
@@ -0,0 +1,21 @@
+from common.models import BaseModel
+from django.db import models
+
+
+class JsonSchema(BaseModel):
```

Comment:
Let's make individual tables by object. This single table could become a pain to navigate